CLDR-13034 kw plurals, fix duplicate samples for 1000.0, 10000.0, 100000.0

eemeli opened the pull request:
[CLDR-13034]

Apologies for not filing an issue about this first, that does not seem possible at the moment?

For now, there's a mismatch between the Cornish (kw) rules and examples, with the decimal numbers 1000.0, 10000.0, and 100000.0 included in the sample list for the `other` category, but being mis-categorised as `two` instead. There is an issue about fixing the rules for this properly ([CLDR-11876](https://unicode.org/cldr/trac/ticket/11876)), but in the interim it would be useful to drop them from the samples, as was done for their integer counterparts in 0481649.

This issue was brought to my attention by @longlho in eemeli/make-plural#13. When parsing the CLDR rules into JavaScript, my code verifies its output using the sample list, and that's currently failing for CLDR v35 Cornish due to this issue.

[CLDR-13034]: https://unicode-org.atlassian.net/browse/CLDR-13034

eemeli commented:
Ah, found the Jira instance via the failing jira-ticket check and managed to create an account. The new issue tracker is apparently not yet mentioned on cldr.unicode.org? Please let me know if I should file a separate issue for this there.

srl295 commented:
@eemeli I've actually fixed this and even redirected from the old to the new tracker.  Also adding a link at the top of the PR, thanks!

srl295 commented:
@eemeli so [CLDR-11078] (if you write it in brackets, it gets linkified) is already closed. Please open a new ticket and reference the prior ones.   Sorry for the bumps, you are exactly outside contributor `#1` on this repo!

srl295 commented:
> When parsing the CLDR rules into JavaScript, my code verifies its output using the sample list, and that's currently failing for CLDR v35 Cornish due to this issue.

Is this possibly a unit test that CLDR ought to have as well? (probably a question for @macchiati )

eemeli commented:
@srl295 I've now added issue [CLDR-13034] and changed the title of this PR, but trying to link the issue to the preceding ones gives me a 401 error. Is there a way I can now force the jira-ticket check to re-process the PR?

srl295 commented:
> Is there a way I can now force the jira-ticket check to re-process the PR?

there is actually a button that does this.

macchiati commented:
This is the wrong fix. Those values are generated by a tool. The fix is to
change the tool, not change the data. This "fix" will be overridden as soon
as we do resolution.

It doesn't need to be backed out, but the ticket needs to stay open and be
assigned to me, for post dsub.

Mark


On Fri, May 17, 2019 at 1:10 PM Steven R. Loomis <notifications@github.com>
wrote:

> Is there a way I can now force the jira-ticket check to re-process the PR?
>
> there is actually a button that does this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/6?email_source=notifications&email_token=ACJLEMD3Z3SOA4C77YKCYTDPV4GMVA5CNFSM4HMJ3DS2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVVX4RA#issuecomment-493583940>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACJLEMEDDTV7CRH2ZUY34FDPV4GMVANCNFSM4HMJ3DSQ>
> .
>

macchiati commented:
ok
Mark


On Thu, May 23, 2019 at 7:22 PM pedberg-icu <notifications@github.com>
wrote:

> *@pedberg-icu* approved this pull request.
>
> Approved as temporary fix, understanding that there will be a more
> complete fix made in tooling and/or data under the same JIRA issue
> CLDR-13034 <https://unicode-org.atlassian.net/browse/CLDR-13034>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/6?email_source=notifications&email_token=ACJLEMANIC6IWJ6BJ25EVDLPW3HEJA5CNFSM4HMJ3DS2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOBZRJOFI#pullrequestreview-241342229>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACJLEMDS26MGC7E43WSINB3PW3HEJANCNFSM4HMJ3DSQ>
> .
>

macchiati commented:
Peter, this is still pending; what do you want to do with it?

srl295 commented:
The ticket here needs to be accepted. Otherwise it is mergeable.

macchiati commented:
This is being closed because the change has to happen in the tooling.
